docs/framework/wcf/samples/transport-udp.md

```diff
@@ -68,7 +68,7 @@ The UDP Transport sample demonstrates how to implement UDP unicast and multicast
 
 -   The``<xref:System.ServiceModel.Channels.ChannelManagerBase> class implements <xref:System.ServiceModel.Channels.CommunicationObject> and provides a unified base class for <xref:System.ServiceModel.Channels.ChannelFactoryBase> and <xref:System.ServiceModel.Channels.ChannelListenerBase>. The <xref:System.ServiceModel.Channels.ChannelManagerBase> class works in conjunction with <xref:System.ServiceModel.Channels.ChannelBase>, which is a base class that implements <xref:System.ServiceModel.Channels.IChannel>.  
   
--   The``<xref:System.ServiceModel.Channels.ChannelFactoryBase> class implements <xref:System.ServiceModel.Channels.ChannelManagerBase> and <xref:System.ServiceModel.Channels.IChannelFactory> and consolidates the `CreateChannel` overloads into one `OnCreateChannel` abstract method.  
+-   The `<xref:System.ServiceModel.Channels.ChannelFactoryBase>` class implements `<xref:System.ServiceModel.Channels.ChannelManagerBase>` and `<xref:System.ServiceModel.Channels.IChannelFactory>` and consolidates the `CreateChannel` overloads into one `OnCreateChannel` abstract method.  
```

Comment:
The fix here is to remove \`\`, not to replace it with a single \`. This should be, "The \<xref:System.ServiceModel.Channels.ChannelFactoryBase> class implements \<xref:..."